ui/client/containers/EspApp.js

```diff
@@ -133,7 +104,7 @@ class EspApp extends React.Component {
                         <Route path={Search.path} component={Search} exact/>
                         <Route path={Signals.path} component={Signals} exact/>
                         <Route path={AdminPage.path} component={AdminPage} exact/>
-                        <Route path={EspApp.path} component={Processes} exact/>
+                        <Redirect from={EspApp.path} to={Processes.path} exact/>
```

Comment:
Why this way?

Reply:
To set URL to /processes. This way we get "Processes" active in menu